FIX: Move ``_stim.<ext>`` specification within the Task Events module (#1750)
bids-standard/bids-specification: src/common-principles.md

            "F": {
                "Description": "Female",
                "TermURL": "https://www.ncbi.nlm.nih.gov/mesh/68005260"
            }
        }
    }
}
```

### Compressed tabular files

Large tabular information such as physiological recordings MAY be stored with
[compressed tab-delineated (TSVGZ) files](glossary.md#tsvgz-extensions).
Rules for formatting plain-text tabular files apply to TSVGZ files with three exceptions:

1.  The contents of TSVGZ files MUST be compressed with
    [gzip](https://datatracker.ietf.org/doc/html/rfc1952).
1.  Compressed tabular files MUST NOT contain a header in the first row
    indicating the column names.
1.  TSVGZ files MUST be accompanied by a JSON file with the same name as their
    corresponding tabular file but with a `.json` extension.

> ???+ warning "Columns of TSVGZ files MUST be defined in the corresponding JSON sidecar and the tabular content MUST NOT include a header line."

Author: Perhaps parsing the html would be easier?

> ???+ warning "Columns of TSVGZ files MUST be defined in the corresponding JSON sidecar and the tabular content MUST NOT include a header line."

Reviewer: nope because we need to be able to turn this into a pdf

> ???+ warning "Columns of TSVGZ files MUST be defined in the corresponding JSON sidecar and the tabular content MUST NOT include a header line."

Author: Are we using pandoc?

> ???+ warning "Columns of TSVGZ files MUST be defined in the corresponding JSON sidecar and the tabular content MUST NOT include a header line."

Reviewer: yeah but also for other contributors I feel that maybe not mixing markdwn + macros + html may be a good thing.

> ???+ warning "Columns of TSVGZ files MUST be defined in the corresponding JSON sidecar and the tabular content MUST NOT include a header line."

Author: https://github.com/chdemko/pandoc-latex-admonition

            "F": {
                "Description": "Female",
                "TermURL": "https://www.ncbi.nlm.nih.gov/mesh/68005260"
            }
        }
    }
}
```

### Compressed tabular files

Large tabular information, such as physiological recordings, MUST be stored with
[compressed tab-delineated (TSVGZ) files](glossary.md#tsvgz-extensions) when

> so established by the specifications.

Reviewer 2: ```suggestion
so established by the specification.
```

Rules for formatting plain-text tabular files apply to TSVGZ files with three exceptions:

1.  The contents of TSVGZ files MUST be compressed with
    [gzip](https://datatracker.ietf.org/doc/html/rfc1952).
1.  Compressed tabular files MUST NOT contain a header in the first row
    indicating the column names.
1.  TSVGZ files MUST be accompanied by a JSON file with the same name as their

>     corresponding tabular file but with a `.json` extension.

Reviewer 2: This seems to ignore the inheritance principle. Are you saying that the inheritance principle doesn't apply to `.tsv.gz` files' metadata?

Reviewer 2: It seems like the admonition's text about column names in the JSON file should go here.

Author: > This seems to ignore the inheritance principle. Are you saying that the inheritance principle doesn't apply to `.tsv.gz` files' metadata?

Can you suggest an alternative phrasing that makes it clear that these files also follow the inheritance principle?

Reviewer 2: Perhaps

```suggestion
1.  TSVGZ files MUST have an associated JSON file that defines the columns in the tabular file.
```
